api/.env.testing

```
@@ -33,6 +36,8 @@ DEMARCHES_SIMPLIFIEES_RIB_VENUE_PROCEDURE_ID_V3=55457
 DEMARCHES_SIMPLIFIEES_RIB_VENUE_PROCEDURE_ID_V4=61168
 DEMARCHES_SIMPLIFIEES_EAC_PROCEDURE_MENJS_CANDIDATE_ID_V2=80270
 DEMARCHES_SIMPLIFIEES_BANK_ACCOUNT_PROCEDURE_ID=80876
+DMS_NEVER_ELIGIBLE_APPLICANT=1
```

```suggestion
DS_NEVER_ELIGIBLE_APPLICANT=1
```
Askip on dit DS, maintenant